### PR TITLE
feat: Stargaze spec champion

### PR DESCRIPTION
**Pull request by BabyScope**
Hi Lava Foundation,

Let me introduce my self I am BabyScope. New with this name in the space but personally almost 14 years in the cryptospace. With Lava i have found my passion back in the space. Helping with rpc providers and also with writing spec to other chains so lava can support more chains then they are present right now. Because lava is capable of greater things. I came here threw Evmos and start growing a name on the lava community and others. I learned a lot from you guys and the community and i hope that i can bring more on the table for you guys. I already made specs for the testnet like UNION , DYMENSION , MOVEMENT , ELYS and KAKAROT. Even helped split up EVMOS with ETHERMINT. So i am capable of maintaining specs as you guys saw with correcting several chains on the testnet. I also edited the COSMOSHUB because there are missing some api's but will add them with the chosen Champion. I hope that i will be chosen and otherwise i will help you guys were i can.

Total Compensation: 4%
Wallet : lava@18rtt3ka0jc85qvvcnct0t7ayq6fva7692k9kvh

### CODE REVIEW

File: Spec/stargaze.json [hunk 1]
```diff
@@ -0,0 +1,798 @@
...
+    "proposal": {
+        "title": "Add Specs: Stargaze",
+        "description": "Adding new specification support for relaying agoric data on stargaze",
+        "specs": [
+            {
+                "index": "STRGZ",
+                "name": "stargaze mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDKFULL"
```
Comment (Yaroms) [inline Spec/stargaze.json:11]: needs to inherit also from cosmossdkv50

Comment (BabyScope) [reply Spec/stargaze.json:11]: https://github.com/lavanet/lava-config/pull/13/commits/23267b06fa9e976fda40c27cbeeb02420b6c3bcc

Fixed here

**Comment by Yaroms** [comment]
all good now i approve

**Comment by Yaroms** [comment]
please make sure you are supporting only relevant apis:
rest:

"/stargaze/tokenfactory/v1/denoms_from_creator/{creator}"
"/stargaze/globalfee/v1/params"
"/stargaze/globalfee/v1/authorizations"
"/stargaze/cron/v1/list-privileged"
"/stargaze/alloc/v1beta1/params"
"/stargaze/globalfee/v1/code_authorization/{code_id}"
"/stargaze/tokenfactory/v1/params"
"/stargaze/tokenfactory/v1/denoms/{denom}/authority_metadata"
"/stargaze/mint/v1beta1/annual_provisions"
"/stargaze/globalfee/v1/contract_authorization/{contract_address}"
"/stargaze/mint/v1beta1/params"
"/stargaze/cron/v1/params"
grpc:

"publicawesome.stargaze.cron.v1.Query/ListPrivileged"
"osmosis.tokenfactory.v1beta1.Query/DenomsFromCreator"
"publicawesome.stargaze.globalfee.v1.Query/CodeAuthorization"
"publicawesome.stargaze.mint.v1beta1.Query/AnnualProvisions"
"publicawesome.stargaze.globalfee.v1.Query/ContractAuthorization"
"publicawesome.stargaze.mint.v1beta1.Query/Params"
"osmosis.tokenfactory.v1beta1.Query/DenomAuthorityMetadata"
"publicawesome.stargaze.globalfee.v1.Query/Params"
"publicawesome.stargaze.cron.v1.Query/Params"
"publicawesome.stargaze.globalfee.v1.Query/Authorizations"
"publicawesome.stargaze.alloc.v1beta1.Query/Params"
"osmosis.tokenfactory.v1beta1.Query/Params"

**Pull request closed without merging**